securedrop/journalist_app/forms.py

```diff
@@ -37,10 +37,17 @@ def name_length_validation(form, field):
             .format(max_chars=Journalist.MAX_NAME_LEN)))
 
 
+def check_invalid_usernames(form, field):
+    invalid_usernames = ['deleted']
```

Comment:
We should keep the canonical list of invalid usernames on `models.Journalist` with other [constraints on usernames](https://github.com/freedomofpress/securedrop/blob/195ead9e16fceb44dbf1ea74f8414231d1ad52fa/securedrop/models.py#L414).